Frontend/src/app/graph-panel/graph/page.tsx

```diff
@@ -0,0 +1,112 @@
+'use client';
+import React, { useEffect } from "react";
+import { Space, Table, Tag, Button, Popconfirm, message } from "antd";
+import type { TableProps } from 'antd';
+import { QuestionCircleOutlined } from '@ant-design/icons';
+import { getGraphList, deleteGraph } from "@/services/graph-service";
+export interface DataType {
+  key: string;
+  name: string;
+  type: string;
+  vertexCount: number;
+  edgeCount: number;
+  status: string;
+}
+
+export default function GraphDetails() {
+  const [graphs, setGraphs] = React.useState<any[]>([]);
+
+  const getGraphsData = async () => {
+    try{
+    const res = await getGraphList();
+    if(res.data){
+      const filteredData: DataType[] = res.data.map((graph: any) => {
+        return {
+          key: graph.idgraph,
+          name: graph.name,
+          type: graph.type,
+          vertexCount: graph.vertexcount,
+          edgeCount: graph.edgecount,
+          status: graph.status,
+        }
+      })
+      setGraphs(filteredData);
+    }
+    }catch(err){
+      message.error("Failed to fetch graphs");
```

Comment:
If this error message is a server side one, shall we add more details from `err` into the message so that it will give more context?

Reply:
yes sure